package.json

```diff
@@ -79,6 +79,7 @@
     "rimraf": "2.7.1",
     "supertest": "3.4.2",
     "travis-cov": "0.2.5",
+    "ts-mocha": "^10.0.0",
```

Comment:
It's for dev purposes, can run tests in TS. No need to build the whole lib to run one test